src/converter-gen/Makefile

```
@@ -2,15 +2,29 @@ include ../testing/make_common.mk
 
 # XML Calabash prefers absolute paths
 test_metaschema:=../testing/models_metaschema.xml
-output_folder:=test_output
+module_path:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
+output_folder:=$(module_path)/test_output
+
+xspec_script=$(realpath $(module_path)/../../support/xslt3-functions/xspec-dev/mvn-saxon-xspec-batch-quiet.sh)
+xspec_ci_script=$(realpath $(module_path)/../../support/xslt3-functions/xspec-dev/mvn-saxon-xspec-batch.sh)
+folder=.
 
 .PHONY: test
 test: smoke-test ## Run all tests
 
 .PHONY: smoke-test
-smoke-test: ## Run all smoke-tests
+smoke-test: ## Apply generators to ../testing/models_metaschema.xml metaschema
```

Minor nit, non-blocking: technically someone can override the default so the documentation can be confusing.

```suggestion
smoke-test: ## Apply generators to test_metaschema
```